docs/src/modules/java/pages/views.adoc

```diff
@@ -29,24 +29,48 @@ Consider an example of a Customer Registry service with a `customer` Value Entit
 
 This example assumes the following `customer` state is defined in a `customer_domain.proto` file:
 
+[.tabset]
+Java::
++
 [source,proto,indent=0]
 .src/main/proto/customer/domain/customer_domain.proto
 ----
 include::example$java-valueentity-customer-registry/src/main/proto/customer/domain/customer_domain.proto[tags=declarations;domain]
 ----
 
+[.tabset]
```

Comment:
I think this is not right (i did the same in the value entity docs) and that it should be a single `[.tabset]` for the two language tabs rather.

Reply:
thanks, I'll change that